Create Blog Post “voice-2022-hackathon-with-vonage-and-symbl-ai” (#2254)
Nexmo/deved-platform: content/blog/en/voice-2022-hackathon-with-vonage-and-symbl-ai.md

---
title: VOICE 2022 Hackathon with Vonage and Symbl.ai
description: Info about the hackathon Vonage and Symbl.ai are putting together for VOICE22
author: christankersley
published: true
published_at: 2022-09-15T18:31:31.793Z
updated_at: 2022-09-15T18:31:31.833Z
category: event
tags:
  - voice-api
  - symbl-ai
  - voice22
comments: true
spotlight: false
redirect: ""
canonical: ""
outdated: false
replacement_url: ""
---

> Vonage is pleased to share that it is partnering with Symbl.ai, a conversational intelligence API, to run a hackathon from September 12, 2022, until October 10, 2022. This means it is running right now as a virtual hackathon, but we will be providing space at VOICE 2022 in Arlington, VA, as part of the VOICE 2022 conference! If that's all you need to know, head over to the [Hackathon page](https://www.voicesummit.ai/2022-hackathon) to register!

Reviewer: Added a hyperlink for Symbl.ai, and a few minor edits to the text. 

```suggestion
Vonage is pleased to share that it is partnering with [Symbl.ai](https://symbl.ai/), a conversational intelligence API, to run a hackathon from September 12, 2022, until October 10, 2022. The hackathon is active right now, and we will be providing space at VOICE 2022 in Arlington, VA, as part of the VOICE 2022 conference! If that's all you need to know, head to the [Hackathon page](https://www.voicesummit.ai/2022-hackathon) to register!
```

## Who is Symbl.ai?

> Symbl.ai is a conversational intelligence API that allows you to process audio, video, and raw text to pull out meaningful information. While you can do text transcription, their API also allows you to find topics that were discussed, questions that were asked, and contextual information about a conversation. Did you just have a meeting? You can send the video to Symbl.ai and have them pull out the important topics and questions to build meeting notes for you.

Reviewer: Tried to tighten this up a bit. 
```suggestion
Symbl.ai is a conversational intelligence API that allows you to process audio, video, and raw text to pull out meaningful information. While you can do text transcription, their API also enables you to find topics discussed, questions asked, and contextual information about a conversation. Suppose you just had a meeting. You can send the video to Symbl.ai and have them pull out the essential topics and questions to build meeting notes for you.
```

> The APIs can be used in real-time through streaming audio to their platform, or you can submit files and text asynchronously and let them process it. Just like Vonage's APIs, they utilize simple HTTP connections to interact with their API and also provide a JavaScript SDK for client applications as well as NodeJS and Python SDKs for server-side applications.

Reviewer: Minor text tweak and added a period to Node.js

```suggestion
The APIs can be used in real-time through streaming audio to their platform, or you can submit files and text asynchronously and let them process it. Just like Vonage's APIs, they utilize simple HTTP connections to interact with their API and provide a JavaScript SDK for client applications and Node.js and Python SDKs for server-side applications.
```

> Find out more about Symbl.ai and its platform at <https://symbl.ai/>.

Reviewer: ```suggestion
Find out more about Symbl.ai and its platform at [https://symbl.ai](https://symbl.ai).
```

## What is VOICE 2022?

> VOICE 2022 brings together top industry leaders and innovators that work with conversation AI. As more and more interactions move to chat and automated systems, conversational AI is becoming a more important component of how companies interact with their customers. Conversational AI can be used to give customers topical, instant information without the need of talking to a human being and can be paired with a variety of software applications like virtual assistants.

Reviewer: ```suggestion
VOICE 2022 brings together top industry leaders and innovators that work with conversation AI. As more and more interactions move to chat and automated systems, conversational AI is becoming a more important component of how companies interact with their customers. Conversational AI can be used to give customers topical, instant information without the need to talk to a human being and can be paired with a variety of software applications like virtual assistants.
```

> VOICE 2022 is being held in Arlington, VA on October 10, 2022, until October 12, 2022. It will be three days filled with well-known companies providing useful sessions and workshops to help you implement conversational AI into your workflows.

Reviewer: ```suggestion
VOICE 2022 takes place in Arlington, VA, on October 10, 2022, until October 12, 2022. It will be three days with well-known companies providing valuable sessions and workshops to help you implement conversational AI into your workflows.
```

> If you are interested in attending VOICE 2022, head over to their website at <https://www.voicesummit.ai/>.

Reviewer: ```suggestion
If you are interested in attending VOICE 2022, head over to their website at [https://www.voicesummit.ai/](https://www.voicesummit.ai/).
```

## VUX World at VOICE

> Vonage is also sponsoring VUX World, a side track on October 11 that runs from 11am until 5pm at VOICE 2022. Come join VUX and kore.ai to get an end-to-end story about call center AI automation. If you have had bad experiences with voice and chatbots in the past, learn from others how you can avoid the pitfalls others have made.

Reviewer: ```suggestion
Vonage also sponsors VUX World, a side track on October 11 that runs from 11 am until 5 pm at VOICE 2022. Join VUX and kore.ai to get an end-to-end story about call center AI automation. If you have had bad experiences with voice and chatbots in the past, learn from others how you can avoid the pitfalls others have made.
```

## What about this Hackathon?

> We want to see what awesome software you can make that utilizes the Vonage Communications Platforms and Symbl.ai. Vonage is a great choice for connecting with your customers while Symbl.ai can automatically provide additional information about those conversations. We want to see how you can use both to potentially make the next great product.

Reviewer: ```suggestion
We want to see what awesome software you can make that utilizes the Vonage Communications Platforms and Symbl.ai. Vonage is a great choice for connecting with your customers, while Symbl.ai can automatically provide additional information about those conversations. We want to see how you can use both to make a great product for your customers.
```
